lib/python-beta/src/bailo/core/client.py

```diff
@@ -260,27 +291,48 @@ def get_files(
             f"{self.url}/v2/model/{model_id}/files",
         ).json()
 
+    def get_download_file(
+        self,
+        model_id: str,
+        file_id: str,
+        localfile_name:str | None = None,
+    ):
+        """
+        Downloads a specific file.
+
+        :param model_id: Unique model ID
+        :param file_id: Unique file ID
+        :param local_file: (optional) The file on the local machine to be copied to
+        :return: A file object
+        """
+        with self.agent.get(f"{self.url}/v2/model/{model_id}/file/{file_id}/download", stream=True) as r:
+            if localfile_name is None:
+                fp = tempfile.NamedTemporaryFile()
```

Comment:
For now I'm just going to use ByteStreams that can be used within a context manager. This will also make them file compatible as well eg. if a user want's to download files
```python
with open("example.txt", "wb") as f:
    release.download(<file-id>, f)
```